### DISCUSSION TITLE
生醫創新創業競賽( ´▽` )ﾉ

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#131] [General]
**新聞內容：**
全國生醫創新創業競賽28日於台北醫學大學發表醫療創新團隊的成果，以下列舉幾項。

1.急性呼吸窘迫症候群(ARDS)患者需要俯臥通氣治療，也要翻身避免褥瘡，過去以「**翻身床**」作為解決辦法，但動輒數十萬元。透過創新比賽的集思廣益，發現可透過兩項**輔具**改善，並讓成本降低至十分之一。

2.台北科技大學提出「**婦嬰AI照護裝置**」，透過收集胎音、腸鳴音，提取特徵進行個人化生物鐘，並透過AI預測嬰兒的照護需求。

3.台北護理健康大學「**盲人智慧手杖**」，盼透過AI技術來辨識，強化導盲杖的功能，如以AI智慧技術來判斷高低差、連結雲端幫忙導航路線、連結公車到站智慧系統等，希望給予盲人更安全的行走環境品質。

4.台北醫學大學「**LumosHospit**」計劃則以大數據與人工智慧，收集並分析臨床實務經驗，輔助醫療人員的決策，如PRS（病房優先辨識系統），協助醫護人員判斷照護優先順序。

**新聞心得：**
生醫創新競賽各團隊推展出的成果，不僅讓大學生能發揮創意提出想法實踐成果，也能讓這些科技運用在往後醫療、生活領域上！我覺得最特別的就是婦嬰AI照護裝置，因為嬰兒很難表達自己的想法，而用AI來預測他們的需求，可以讓家長減輕不少壓力。

**新聞來源：**
https://udn.com/news/story/7266/5922962

**Comment by wh20502** [comment]
看到ai連盲人都能提供協助，是否以後有機會出現穿戴使耳機ai廣播系統，可以隨時播報周圍狀況，讓盲人也能夠在街道上以耳朵建構出周昭環境。

**Comment by minniewei** [comment]
種種科技讓生活變得更美好!!!期待未來生醫的發展!!!

**Comment by Shiba010** [comment]
AI在生醫的發展看起來會是未來人類生活的福音！
